arviz/plots/backends/bokeh/densityplot.py

```diff
@@ -90,8 +89,18 @@ def plot_density(
                 outline,
                 shade,
                 axis_map[label],
-                data_label=data_label,
             )
+            if data_label is not None:
+                if axis_map[label] in legend_items.keys():
+                    legend_items[axis_map[label]].append((data_label, plotted))
+                else:
+                    legend_items[axis_map[label]] = []
+                    legend_items[axis_map[label]].append((data_label, plotted))
```

Comment:
Could legend items be a defaultdict?

Reply:
https://docs.python.org/3/library/collections.html#collections.defaultdict

Reply:
Yes, it will make the code neat. Thanks, for the suggestion.